pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitTestUtil.java

```diff
@@ -513,5 +516,16 @@ public void setEsIndex(String esIndex) {
         new DDPInstanceDao().updateEsParticipantIndex(ddpInstanceId, esIndex);
     }
 
+
+    /**
+     * reads the test config and sets it as what dsm uses as config.
+     * This is needed for the tests that use values from config/secret to run
+     * */
+    public static void setDsmConfig() {
+        ConfigManager configManager = ConfigManager.getInstance();
+        Config cfg = configManager.getConfig();
+        new DSMConfig(cfg);
```

Comment:
Because internally newing up DSMConfig actually changes static state, it's possible that this may run into race conditions during tests, where some tests attempt to set DSMConfig.config differently between local envs vs circle.  If in EventConfig we set things like so:

``` 
long initialInterval = ConfigManager.getInstance().getConfig().getInt(ApplicationConfigConstants.EVENT_RETRY_INTERVAL_MS); // base delay in
    double multiplier = ConfigManager.getInstance().getConfig().getDouble(ApplicationConfigConstants.EVENT_RETRY_MULTIPLIER); // exponential backoff
```
then I think we can avoid this risk.

Reply:
That solved the problem! 

The problem here was that the config for DSM tests does not come from the dsm secret, it comes from the testing-inmemory-db while DSM is set to use DSMConfig that is created upon server start. `ConfigManager.getInstance()` was a great way to handle this, thanks